packages/flutter/lib/src/widgets/editable_text.dart

```diff
@@ -2727,11 +2721,13 @@ class EditableTextState extends State<EditableText> with AutomaticKeepAliveClien
     _showCaretOnScreenScheduled = true;
     SchedulerBinding.instance.addPostFrameCallback((Duration _) {
       _showCaretOnScreenScheduled = false;
-      if (_currentCaretRect == null || !_scrollController.hasClients) {
+      final RenderEditable? renderEditable =
+          _editableKey.currentContext?.findRenderObject() as RenderEditable?;
```

Comment:
It's probably not safe to call `findRenderObject` in a post frame callback. `RenderEditable` may have been disposed.

Reply:
In that case would this just be null or would it actually be an error?

Reply:
From my tests it looks like currentContext will be null. This is why I used the null-aware operator here instead of using the renderEditable getter. IIRC this was also seen when running widget tests, particularly those that force removing the TextField from the tree by calling tester.pumpWidget() with some other widget.